{{ cookiecutter.project_slug }}/src/helpers/health_check_middleware.py

```diff
@@ -0,0 +1,13 @@
+from django.conf import settings
+
+
+class HealthCheckMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+    
+    def __call__(self, request):
+        if request.path.replace("/", "") == "health_check":
+            request.META["HTTP_HOST"] = "localhost"
+            if settings.ALLOWED_HOSTS and not settings.DEBUG:
+                request.META["HTTP_HOST"] = settings.ALLOWED_HOSTS[0]
```

Comment:
This will break if `ALLOWED_HOST` is []. Use `[:1]` instead.

Reply:
I'm validating if `ALLOWED_HOST` is empty here:

```py
if settings.ALLOWED_HOSTS
```

Reply:
Ah right. But either way, you cannot be sure the first `ALLOWED_HOST` is the current used host. I would change this and not really depend on the ALLOWED_HOST. Instead, I would get the current request.host, and check if its in the ALLOWED_HOSTS.